manifests/init.pp

```diff
@@ -82,19 +84,20 @@
     destination        => "${jar_path}/${jar_name}",
     timeout            => $download_timeout,
     nocheckcertificate => $nocheckcertificate,
-    execuser           => $user,
```

Comment:
This is ineeded on posix systems as there's no guarantee about the root users umask.  It might make sense to switch to `nanliu/archive` if it works better with windows.